### ISSUE TITLE
Ethereum Foundation is a potential source of traffic

### ISSUE THREAD

**Issue by rihp** [dOrgTech/landing-page#72]
We can link dorg.tech in this link, under the DAOs section https://ethereum.org/en/community/

**Comment by rihp** [comment]
The repo is here: 
We could propose PR
https://github.com/ethereum/ethereum-org-website

**Comment by rihp** [comment]
Edit here : https://github.com/rihp/ethereum-org-website/commit/7995fb7674a7b35130de03250197bc3df63a999c

**Comment by orishim** [comment]
@rihp it doesn't look like you ever made the PR

**Comment by rihp** [comment]
@orishim yeah, I was waiting for your confirmation. I'll revisit this, update copy, and set PR

```
- [dOrg](https://dOrg.tech) [@dOrg_tech](https://twitter.com/dOrg_tech) - Freelancer Web3 development collective working as a DAO 
```

**Comment by rihp** [comment]
submitted PR
https://github.com/ethereum/ethereum-org-website/pull/1723
